framework/lib/components/select/types.ts

```diff
@@ -20,8 +20,9 @@ export interface SelectProps<T>
   ChakraReactSelectProps<T, boolean, GroupBase<T>>,
   'onChange' | 'value'
   > {
-  /** Whatever is currently selected by the select will be controlled by value prop */
-  value?: Option | Option[]
+  /** Whatever is currently selected by the select will be controlled by value prop, if value is
+   *  of type string then it will automatically pair it up with matching object from options list */
+  value?: T | T[] | string | string[]
```

Comment:
Is it possible to get those to be selected conditionally? (based on props)

Reply:
Could you elaborate on how this might be done? Not sure exactly what you have in mind, you could make another generic prop and set it equal to value prop I guess?

Reply:
Let's discuss it when you're in the office :)

Reply:
@olchyk98 plz re-review this PR